code/modules/projectiles/sentries.dm

```diff
@@ -440,3 +454,25 @@
 
 		distance = buffer_distance
 		return nearby_target
+
+
+/obj/machinery/deployable/mounted/sentry/buildasentry
+
+/obj/machinery/deployable/mounted/sentry/buildasentry/examine(mob/user)
```

Comment:
you can fill the void here with something like
name = "BROKEN SENTRY"
descc = "report this on github"